apis/apis.json

```diff
@@ -576,7 +576,7 @@
 
   {
     "id": "Google.Cloud.EntityFrameworkCore.Spanner",
-    "version": "1.0.0-alpha04",
+    "version": "1.0.0-alpha05",
```

Comment:
I'm very tempted to remove this as a project. (No change required now, just thinking...)